packages/ripple-keypairs/.eslintrc.js

```diff
@@ -56,5 +56,6 @@ module.exports = {
     'eslint-comments/require-description': 'off',
     'no-shadow': 'off',
     'multiline-comment-style': 'off',
+    '@typescript-eslint/no-require-imports': 'off',
```

Comment:
I'd suggest that we leave this at `warn` instead of `off`, but IMO this should be ignored every time we need to do this, instead of with a global ignore.

Reply:
Warn is usually treated as a failure I"m pretty sure in our code.

Also this syntax is correct so I'm not sure what value this rule brings?